### PR TITLE
Enable submit on Staff View auth page

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR connects the Staff View auth page to the new POST request, and updates the Staff View confirmation page to use actual data. Staff View works!

===

Resolves #XXX

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: src/client/pages/StaffViewAuthPage/index.js [hunk 1]
```diff
@@ -91,35 +89,7 @@ function StaffViewAuthPage(props) {
       .then((response) => response.json())
       .then((data) => {
         setUserData(data);
-        if (data.authToken) {
```
Comment (sharonwarner) [inline src/client/pages/StaffViewAuthPage/index.js:94]: I believe we still need to update line 78 to pull the staff admin login.

File: src/client/pages/StaffViewConfirmationPage/index.js [hunk 2]
```diff
@@ -72,9 +74,9 @@ function StaffViewConfirmationPage(props) {
             <span className="badge completed">
               {t("staff-view-confirmation.completed.status")}
             </span>
-            {/* TODO(kalvin): make last name real */}
             <p>
-              {t("staff-view-confirmation.completed.last-name")}
+              {t("staff-view-confirmation.completed.last-name")} +
+              userData.lastName
```
Comment (sharonwarner) [inline src/client/pages/StaffViewConfirmationPage/index.js:79]: It looks like this isn't showing up correctly for some reason.

Comment (kalvinwang) [reply src/client/pages/StaffViewConfirmationPage/index.js:79]: whoops, i’ve been using renderNonTransContent for retrocert pages with Trans components, so those snapshots aren’t actually rendering inside some components :grimacing: fixing…

File: src/client/pages/StaffViewAuthPage/index.js (hunk 1 above)
Comment (kalvinwang) [reply src/client/pages/StaffViewAuthPage/index.js:94]: oops, thanks!

**Pull request merged**